test/e2e/vite.config.mts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import { alias } from '../../vitest.shared.mjs';
+
+export default defineConfig({
+  build: {
+    outDir: 'build',
+  },
+  plugins: [
+    react(),
+    {
+      name: 'replace-code',
+      enforce: 'post',
+      async transform(code) {
+        return code
+          .replaceAll('DISABLE_CHANCE_RANDOM', 'true')
+          .replaceAll('LICENSE_DISABLE_CHECK', 'true');
+      },
+    },
+  ],
+  resolve: {
+    alias,
+  },
+  test: {
+    globals: true,
+    testTimeout: 20000,
+    hookTimeout: 20000,
```

Comment:
We also seem to be setting timeouts in line 112 of `index.test.ts`. Would it make sense to move all the timeout set up here so that it's centralized?